integration/pmem/pmem_test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../.ci/lib.sh"
+image="fedora"
+payload="tail -f /dev/null"
+container_name="test-pmem"
+osbuilder_repository="github.com/kata-containers/osbuilder"
+osbuilder_repository_path="${GOPATH}/src/${osbuilder_repository}"
+test_directory_name="test_pmem1"
+test_directory="/tmp/${test_directory_name}"
```

Comment:
We should always use `mktemp` unless there is a documented reason why not to:

```sh
test_directory=$(mktemp -d --suffix="${test_directory_name}")
```

You can then drop the `mkdir` below.